.gitignore

```diff
@@ -11,6 +11,7 @@ spec.opts
 .ruby-version
 .java-version
 .ruby-gemset
+.rakeTasks
```

Comment:
Any idea what is creating a `.rakeTasks` file and/or what's in it - I have no issues with that when running tests.

Reply:
I can remove it if you want.